azurerm/resource_arm_container_group.go

```diff
@@ -279,6 +279,48 @@ func resourceArmContainerGroup() *schema.Resource {
 				},
 			},
 
+			"log_analytics": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"workspace_id": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.NoEmptyStrings,
+						},
+						"workspace_key": {
+							Type:         schema.TypeString,
+							Required:     true,
+							Sensitive:    true,
+							ForceNew:     true,
+							ValidateFunc: validate.NoEmptyStrings,
+						},
+						"log_type": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								string(containerinstance.ContainerInsights),
+								string(containerinstance.ContainerInstanceLogs),
+							}, true),
```

Comment:
Could we use case-sensitive comparison for enumeration values? This is a new pattern (for your reference: https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_dev_test_linux_virtual_machine.go#L66) to deal with enumerations.
```suggestion
							}, false),
```